server/package.json

```diff
@@ -5,16 +5,19 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "start": "node index.ts",
+    "start": "npm run serve",
     "compile": "npx tsc",
-    "pre-build": "rm -rf ../dist",
-    "build": "tsc && npm run lint",
-    "post-build": "cd ../dist && npm install",
+    "reset": "rm -Rf dist && rm -Rf node_modules",
+    "build": "npx tsc && npm run lint",
+    "post-build": "cd dist/ && npm install",
     "lint": "npx tsc --noEmit && eslint \"**/*.{js,ts}\" --quiet --fix",
-    "copy-static": "mkdir -p ../dist/static && cp -R static/ ../dist/static/ && cp package* ../dist", 
-    "build-prod": "npm run pre-build && npm run build && npm run copy-static && npm run post-build",
-    "build-dist": "npm run pre-build && npm run compile && npm run copy-static && npm run post-build",
-    "serve": "node ../dist/index.js"
+    "copy-static": "cp -R public dist/ && cp package* dist/",
+    "build-local": "npm install --production=false && npm run build && npm run copy-static && npm run post-build",
+    "build-dist": "npm i typescript && npm run compile && npm run copy-static && npm run post-build",
```

Comment:
I'm not too fond of chaining commands in here. This is only used in the Dockerfile, right? Why not list all commands there?

Reply:
Build-prod is for local builds where you run, all the steps including linting compilation, more checks to be added if needed. 
Build-dist is yes for running inside the docker file but since it uses npm commands i thought it would be easier to keep the context inside package.json